src/logger/loggerApi.c

```diff
@@ -66,25 +65,16 @@
 #include <stdlib.h>
 #include <string.h>
 
-/* Max number of PIDs that can be specified in the setOBD2Cfg message */
-#define MAX_OBD2_MESSAGE_PIDS 10
+/* Max number of channels that can be specified in the setOBD2Cfg message */
+#define MAX_OBD2_MESSAGE_PIDS 4
```

Comment:
This is a constraint on how many PIDs at a time can be sent in one call.  IIRC the app currently sends all of them in a group, but there's a limitation in the current app where you can't set more than 10 due to JSON message size. 

So, we need to send N at a time, this is the limit of how many PIDs can be sent in one "set" API call. 

However, right now the 1.9.0 app sends PIDs one by one in discrete 'set' messages.